eZ/Publish/SPI/Search/Document.php

```diff
@@ -31,6 +31,27 @@ class Document extends ValueObject
     public $languageCode;
 
     /**
+     * Denotes that document's translation is always available
```

Comment:
From the Solr code above it seems to actually be:
`Denotes that document's translation is the main translation and it is always available`